ts/visitor.ts

```diff
@@ -191,6 +191,12 @@ export const visitor = {
   },
   Function: {
     enter(path: NodePath<t.Function>, st: S) {
+      if (path.node.params.length &&
+          path.node.params[path.node.params.length - 1].type === 'RestElement') {
+        st.elem.error(path, `The rest parameter is not supported.`);
```

Comment:
Is it not supported, or disallowed? Would be good to be open.

Reply:
>Is it not supported, or disallowed?

Well I guess it's disallowed because it's not supported (?). I can always tweak the message.

>Would be good to be open.

The issue was indifferent; "Disallow (or support)...". This was easier. Although, if support for the rest parameter is something that should be in EJS, then it can be added.
@arjun: Do you want to weigh-in?

Reply:
I'm happy with "not supported", since that is what is actually happening.

Reply:
Alright, I think we can leave the changes as-is. If we want to add rest parameter support in the future, then we can always do so. This change will at least mask the arity problem with rest that currently exists, so to prevent program crashes.